jp/src/edu/hillel/lesson21/DataBaseConnection.java

```diff
@@ -0,0 +1,30 @@
+package edu.hillel.lesson21;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DataBaseConnection {
+    private static final String URL = "jdbc:mysql://localhost:3306/testing";
+    private static final String USER = "root";
+    private static final String PASSWORD = "root";
+
+    public Connection getConnection() {
+        try {
+            return DriverManager.getConnection(URL, USER, PASSWORD);
+        } catch (SQLException e) {
+            e.printStackTrace();
+            return null;
+        }
+    }
+
+    public void close(Connection connection) {
```

Comment:
Please, implement AutoClosable interface by this class and override close method to close connection properly

Reply:
Changed